Test/Unit/TestHelper.php

```diff
@@ -19,7 +19,9 @@ class TestHelper extends TestCase
      */
     public static function getReflectedClass($class)
     {
-        if (is_subclass_of($class, 'PHPUnit_Framework_MockObject_MockObject')) {
+        if (is_subclass_of($class, '\PHPUnit\Framework\MockObject\MockObject')) {
+            return new ReflectionClass(get_parent_class($class));
+        } else if (is_subclass_of($class, 'PHPUnit_Framework_MockObject_MockObject')) {
```

Comment:
as the result is the same for both conditions you can remove the `else if` clause and put both checks into `if`

Reply:
Done